src/WebJobs.Script/Utility.cs

```diff
@@ -681,6 +681,17 @@ public static bool CheckAppOffline(IEnvironment environment, string scriptPath)
             return false;
         }
 
+        public static void ExecuteAfterDelay(Action targetAction, TimeSpan delay, CancellationToken cancellationToken = default)
+        {
+            Task.Delay(delay).ContinueWith(_ =>
```

Comment:
Please make sure the cancellation token is honored by passing it into the delay call.

Reply:
oh right, I missed that somehow